Cargo.toml

```diff
@@ -56,7 +56,7 @@ log = "0.4"
 png = "0.16"
 winit = { version = "0.22.1", features = ["web-sys"] }
 rand = { version = "0.7.2", features = ["wasm-bindgen"] }
-bytemuck = "1"
+bytemuck = { version = "1", features = ["derive"] }
```

Comment:
I assume this feature is only available since some `1.x` minor version?

Reply:
Good point! fixed.